libpldmresponder/platform_config.cpp

```diff
@@ -151,15 +151,17 @@ std::optional<std::string> Handler::getSysSpecificJsonDir(
         return std::nullopt;
     }
 
-    for (const auto& dirEntry : std::filesystem::directory_iterator{dirPath})
+    for (const auto& compatibleName : dirNames)
     {
-        if (dirEntry.is_directory())
+        for (const auto& dirEntry : std::filesystem::directory_iterator{dirPath})
         {
-            const auto sysDir = dirEntry.path().filename().string();
-            if (std::find(dirNames.begin(), dirNames.end(), sysDir) !=
-                dirNames.end())
+            if (dirEntry.is_directory())
             {
-                return sysDir;
+                const auto sysDir = dirEntry.path().filename().string();
```

Comment:
This sysDir is not used anywhere, can we get rid of it?

Reply:
Ohh forgot to delete it. Will update and repush.